test/test_pymongo_auth_aws.py

```diff
@@ -52,6 +56,117 @@ def test_region(self):
         self.assertRaises(PyMongoAuthAwsError, _get_region, '.first.second')
         self.assertRaises(PyMongoAuthAwsError, _get_region, 'first.second.')
 
+    def test_aws_temp_credentials_env_variables(self):
+        os.environ['AWS_ACCESS_KEY_ID'] = 'foo'
+        os.environ['AWS_SECRET_ACCESS_KEY'] = 'bar'
+        creds = _aws_temp_credentials()
+        del os.environ['AWS_ACCESS_KEY_ID']
+        del os.environ['AWS_SECRET_ACCESS_KEY']
+        assert creds.username == 'foo'
```

Comment:
Why use assert instead of self.assertEqual?

Reply:
I'm used to writing `pytest`-style tests :smile:, updated